test/is-primitive.spec.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+var expect = require('chai').expect;
+var isPrimitive = require('../src/is-primitive').isPrimitive;
+
+describe('isPrimitive', function () {
+  describe('should be primitive', function () {
+    it('boolean', function () {
+      expect(isPrimitive(true)).to.be.true;
+    });
+
+    it('number', function () {
+      expect(isPrimitive(1)).to.be.true;
+    });
+
+    it('string', function () {
+      expect(isPrimitive('a string')).to.be.true;
+    });
+
+    it('null', function () {
+      expect(isPrimitive(null)).to.be.true;
+    });
+
+    it('undefined', function () {
+      expect(isPrimitive(undefined)).to.be.true;
+    });
+
+    it('NaN', function () {
+      expect(isPrimitive(NaN)).to.be.true;
+    });
+
+    it('Infinite', function () {
+      expect(isPrimitive(Infinity)).to.be.true;
```

Comment:
I'm not actually sure if `NaN` and `Infinity` are considered primitive.
I think what you want:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures